### PR TITLE
API endpoint that enables the deletion of a specific event that is not past.

### LINKED ISSUE #137: API: Create an api to delete/cancel an event that is not past
Create an API endpoint that enables the deletion of a specific event that is not past. 

The endpoint should take as parameters the event_id and the user_id and consider that only Admin role can delete events. It should filter the date of the event so it makes sure that it's only available for future events.

This API will be used on this component - #135 

The endpoint has to be documented on Swagger

### PR DESCRIPTION
**Pull request by AkoAyukngat**
# Description

This API  will delete an event that is in the future and the deletion process can only be done by an Admin. Or else it will throw an error 403 if the person trying to perform the delete is not an admin.

Fixes #137

# How to test?

You can use swagger to test this API

# Checklist

- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by bytewiz — CHANGES_REQUESTED** [review]
Looks to me from the description that you have some unrelated changes.

I only see this PR should add deletion event but it also contains creating feedback

File: src/server/api/routes/api-router.js [hunk 1]
```diff
@@ -45,9 +46,15 @@ router.use('/topics', TopicsRouter);
 
 router.use('/skills', skillsRouter);
 
-router.use('/events', eventsRouter);
 router.use('/groups', groupsRouter);
 
+router.use('/feedbacks', feedbacksRouter);
+
+router.use('/events', eventsRouter);
```
Comment (tpetrina) [inline src/server/api/routes/api-router.js:53]: Why did you move this?

File: src/server/api/routes/api-router.js [hunk 2]
```diff
@@ -45,9 +46,15 @@ router.use('/topics', TopicsRouter);
...
+
+router.use('/', function (req, res) {
+  res.redirect(`${process.env.API_PATH}/documentation`);
+});
```
Comment (tpetrina) [inline src/server/api/routes/api-router.js:57]: Can you remove these three lines - they are not needed here.

File: src/server/api/routes/events.router.js [hunk 3]
```diff
@@ -57,4 +57,39 @@ router.get('/:id', (req, res, next) => {
...
+ *        name: eventId
+ *        description: ID of the event to delete.
+ *    responses:
+ *      200:
+ *        description: Event deleted
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.delete('/:userId/:eventId', (req, res) => {
+  console.log(req.params);
```
Comment (tpetrina) [inline src/server/api/routes/events.router.js:81]: Remove console.log.

File: src/server/app.js [hunk 4]
```diff
@@ -46,17 +46,17 @@ app.use((err, req, res) => {
...
+//   res.status(404).send("Sorry can't find that!");
+// });
 
 // If "/api" is called, redirect to the API documentation.
-app.use('/api', function (req, res) {
-  res.redirect(`${process.env.API_PATH}/documentation`);
-});
+// app.use('/api', function (req, res) {
+//   res.redirect(`${process.env.API_PATH}/documentation`);
+// });
```
Comment (tpetrina) [inline src/server/app.js:59]: Can you please undo changes in this file?

**Review by tpetrina — CHANGES_REQUESTED** [review]
Good start, but extra code should be removed.

File: src/server/api/routes/api-router.js (hunk 1 above)
Comment (AkoAyukngat) [reply src/server/api/routes/api-router.js:53]: Hi Toni, I moved it when I was trying to fix the Internal Server Error I had. I t worked for me but maybe I could I have undo the changes before submission.

File: src/server/app.js (hunk 4 above)
Comment (AkoAyukngat) [reply src/server/app.js:59]: Yes I will

File: src/server/api/routes/api-router.js (hunk 1 above)
Comment (tpetrina) [reply src/server/api/routes/api-router.js:53]: Yes, please undo them.

**Comment by AkoAyukngat** [comment]
so much conflicts so decided to close and make a new pull request
